### ISSUE TITLE
Application doesn't work when follwing the README

### ISSUE THREAD

**Issue by simonc** [empress/guidemaker#17]
Hello there 👋 

I've been trying to follow the instructions provided in the README and the application breaks on first launch.

Here is a trace of the setup:

```
$ node --version
v8.11.1

$ npm --version
6.9.0

$ npm install -g ember-cli
/Users/simonc/.nvm/versions/node/v8.11.1/bin/ember -> /Users/simonc/.nvm/versions/node/v8.11.1/lib/node_modules/ember-cli/bin/ember
+ ember-cli@3.9.0
updated 1 package in 12.27s

$ ember new pdfmonkey-docs
installing app
  create .editorconfig
  create .ember-cli
  create .eslintignore
  create .eslintrc.js
  create .template-lintrc.js
  create .travis.yml
  create .watchmanconfig
  create README.md
  create app/app.js
  create app/components/.gitkeep
  create app/controllers/.gitkeep
  create app/helpers/.gitkeep
  create app/index.html
  create app/models/.gitkeep
  create app/resolver.js
  create app/router.js
  create app/routes/.gitkeep
  create app/styles/app.css
  create app/templates/application.hbs
  create app/templates/components/.gitkeep
  create config/environment.js
  create config/optional-features.json
  create config/targets.js
  create ember-cli-build.js
  create .gitignore
  create package.json
  create public/robots.txt
  create testem.js
  create tests/helpers/.gitkeep
  create tests/index.html
  create tests/integration/.gitkeep
  create tests/test-helper.js
  create tests/unit/.gitkeep
  create vendor/.gitkeep
npm: Installed dependencies
Successfully initialized git.

$ cd pdfmonkey-docs

$ ember install guidemaker guidemaker-default-template
npm: Installed guidemaker and guidemaker-default-template
installing guidemaker
? Overwrite app/index.html? Yes, overwrite
  overwrite app/index.html
  create guides/getting-started/editing.md
  create guides/getting-started/index.md
  create guides/index.md
  create guides/pages.yml
  create node-tests/link-checker.js
  create node-tests/toc-checker.js
  remove app/templates/application.hbs
  install packages guidemaker-link-checker, guidemaker-toc-checker, mocha
npm: Installed guidemaker-link-checker, guidemaker-toc-checker and mocha
  install addons prember, ember-cli-fastboot
npm: Installed prember and ember-cli-fastboot
Installed addon package.
installing guidemaker-default-template
  install addon ember-cli-google-fonts
npm: Installed ember-cli-google-fonts
Installed addon package.
Installed addon package.
```

Then when I start the server and visit the app on localhost:4200 I get this:

```
$ npm start

> pdfmonkey-docs@0.0.0 start /Users/simonc/Documents/PDFMonkey/pdfmonkey-docs
> ember serve

'require' is imported by ../../../../../../../var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/cache-128-rollup/build/-private/utils/fetch.js, but could not be resolved – treating it as an external dependency

Build successful (16726ms) – Serving on http://localhost:4200/

Slowest Nodes (totalTime => 5% )              | Total (avg)
----------------------------------------------+---------------------
Babel: ember-data (2)                         | 2897ms (1448 ms)
Babel: pdfmonkey-docs (3)                     | 2136ms (712 ms)
Babel: ember-test-helpers (2)                 | 1737ms (868 ms)
Rollup (1)                                    | 1122ms
Babel: ember-power-select (2)                 | 1106ms (553 ms)
Babel: ember-basic-dropdown (2)               | 917ms (458 ms)

App is being served by FastBoot
DEBUG: -------------------------------
DEBUG: Ember      : 3.9.1
DEBUG: Ember Data : 3.9.2
DEBUG: -------------------------------
DEPRECATION: The <pdfmonkey-docs@adapter:version::ember198>#fastboot computed property was just overriden. This removes the computed property and replaces it with a plain value, and has been deprecated. If you want this behavior, consider defining a setter which does it manually. [deprecation id: computed-property.override] See https://emberjs.com/deprecations/v3.x#toc_computed-property-override for more details.
        at logDeprecationStackTrace (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/@ember/debug/lib/deprecate.js:105:1)
        at HANDLERS.(anonymous function) (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/@ember/debug/lib/handlers.js:23:1)
        at raiseOnDeprecation (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/@ember/debug/lib/deprecate.js:132:1)
        at HANDLERS.(anonymous function) (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/@ember/debug/lib/handlers.js:23:1)
        at invoke (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/@ember/debug/lib/handlers.js:35:1)
        at deprecate (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/@ember/debug/lib/deprecate.js:170:1)
        at ComputedProperty.clobberSet (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/@ember/-internals/metal.js:3038:1)
        at ComputedProperty.set (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/@ember/-internals/metal.js:3023:1)
        at initialize (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/@ember/-internals/runtime/lib/system/core_object.js:61:1)
        at Function.create (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/@ember/-internals/runtime/lib/system/core_object.js:611:1)
        at FactoryManager.create (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/@ember/-internals/container.js:554:1)
        at instantiateFactory (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/@ember/-internals/container.js:363:1)
        at _lookup (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/@ember/-internals/container.js:290:1)
        at Container.lookup (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/@ember/-internals/container.js:134:1)
        at Class.lookup (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/@ember/-internals/runtime/lib/mixins/container_proxy.js:78:1)
        at Class.adapterFor (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/addon-tree-output/ember-data/-private.js:12642:1)
        at Class._flushPendingFetchForType (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/addon-tree-output/ember-data/-private.js:10897:1)
        at Map.forEach (<anonymous>)
        at Class.flushAllPendingFetches (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/addon-tree-output/ember-data/-private.js:10890:1)
        at invokeWithOnError (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/backburner.js:350:1)
        at Queue.flush (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/backburner.js:233:1)
        at DeferredActionQueues.flush (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/backburner.js:437:1)
        at Backburner._end (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/backburner.js:1013:1)
        at Backburner._boundAutorunEnd (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/backburner.js:653:1)
        at <anonymous>
        at process._tickCallback (internal/process/next_tick.js:188:7)
Error while processing route: index (0 , _private.fetch) is not a function TypeError: (0 , _private.fetch) is not a function
    at Class._fetchRequest (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/addon-tree-output/ember-data/adapters/rest.js:944:1)
    at Class.ajax (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/addon-tree-output/ember-data/adapters/rest.js:887:1)
    at Class.findRecord (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/addon-tree-output/ember-data/adapters/rest.js:437:1)
    at Ember.RSVP.Promise.resolve.then (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/addon-tree-output/ember-data/-private.js:7033:1)
    at tryCatcher (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/rsvp.js:334:1)
    at invokeCallback (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/rsvp.js:507:1)
    at /var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/rsvp.js:572:1
    at /var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/ember-testing/lib/ext/rsvp.js:17:1
    at invokeWithOnError (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/backburner.js:350:1)
    at Queue.flush (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/backburner.js:233:1)
    at DeferredActionQueues.flush (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/backburner.js:437:1)
    at Backburner._end (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/backburner.js:1013:1)
    at Backburner._boundAutorunEnd (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/backburner.js:653:1)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
TypeError: (0 , _private.fetch) is not a function
    at Class._fetchRequest (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/addon-tree-output/ember-data/adapters/rest.js:944:1)
    at Class.ajax (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/addon-tree-output/ember-data/adapters/rest.js:887:1)
    at Class.findRecord (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/addon-tree-output/ember-data/adapters/rest.js:437:1)
    at Ember.RSVP.Promise.resolve.then (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/addon-tree-output/ember-data/-private.js:7033:1)
    at tryCatcher (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/rsvp.js:334:1)
    at invokeCallback (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/rsvp.js:507:1)
    at /var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/rsvp.js:572:1
    at /var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/ember-testing/lib/ext/rsvp.js:17:1
    at invokeWithOnError (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/backburner.js:350:1)
    at Queue.flush (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/backburner.js:233:1)
    at DeferredActionQueues.flush (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/backburner.js:437:1)
    at Backburner._end (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/backburner.js:1013:1)
    at Backburner._boundAutorunEnd (/var/folders/90/y7lt76pd6jj7sp0kr3mcv5600000gn/T/broccoli-53391zWeqKFM4EuB2/out-357-broccoli_merge_trees/assets/backburner.js:653:1)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

Is there anything I'm doing wrong? A dependency I should add?

Thanks! <3

**Comment by mansona** [comment]
hey @simonc 👋 

So I did a bit of investigation and it seems like this is an error upstream in ember-data 🙈 we have been having some issues with that project in recent releases so thank you for bringing this to my attention

Right now the workaround would be to fix your `ember-data` version to `3.9.1` by replacing the line: 

```
"ember-data": "~3.9.0",
```

with 

```
"ember-data": "3.9.1",
```

and running `npm install` again.

Let me konw if that helps 👍

**Comment by simonc** [comment]
Hi @mansona,

It worked! Thanks! 🧡

I still have the deprecation warning but that expected. I can move forward now. So cool 😊
